tests/integration/test_api_resources.py

```diff
@@ -107,6 +109,35 @@ def test_successful_upload_from_url_sync_with_filename(self):
         self.assertEqual(file.filename(), 'meh.png')
 
 
+@unittest.skipIf(not os.environ.get("SIGNED_UPLOADS_PUBKEY"),
+                 'Signed uploads project not configured')
+class SignedUploadsTest(RestAPITestCase):
+    def setUp(self):
+        conf.pub_key = os.environ.get("SIGNED_UPLOADS_PUBKEY")
+        conf.secret = os.environ.get("SIGNED_UPLOADS_SECKEY")
+        conf.signed_uploads = True
+
+    def tearDown(self):
+        conf.pub_key = None
+        conf.secret = None
+        conf.signed_uploads = False
```

Comment:
I think we could use mock's for this. `mock.patch.object` as class decorators and there will be no need to add `tearDown` and `setUp` methods.

Reply:
may be later :)